{{cookiecutter.project_slug}}/requirements/test.txt

```diff
@@ -1,18 +0,0 @@
-# Test dependencies go here.
```

Comment:
why remove them?

Reply:
What we call test environment is essentially a superset of a dev env with an extra few infrastructural dependencies (e.g. `pytest-sugar`).

Reply:
Okay. It would be better to do deletions like that in a single PR.

The current diff makes it super hard to see what's going on since every line has changed :D

Reply:
@jayfk yes, indeed.